content/troubleshooting/java/application/Java-EffectsOnAppPerformance.md

```diff
@@ -4,18 +4,17 @@ description: "Tips to improve app performance with the Java agent"
 tags: "troubleshoot java agent performance latency startup"
 -->
 
-As you might expect, Contrast's analysis does make your app run a little slower. The good news is it's generally not enough for anyone to complain about, and the results are definitely worth it.
+Contrast's platform has been designed from the ground up to provide maximum value to customers across the SDLC, while minimizing overhead. The following sections discuss some general considerations, as well as considerations specific to Contrast Assess and Contrast Protect.
 
+## General Considerations
```

Comment:
If you want to group "Startup Time", "Performance Tuning Hints", and "Memory" under the "General Considerations" heading, you'll need to increase their heading depth i.e. `### Startup Time`